README.md

```diff
@@ -6,26 +6,26 @@ Very simple package that aims at adding some color to Your life.
 - Add color codes to text (uses color numbers)
 ``` python
 >>> from incolor import incolor
->>> incolor(4, 'I\'m blue da ba dee')
+>>> incolor('I\'m blue da ba dee', fg=4)
 "\x1b[34mI'm blue da ba dee\x1b[0m"
 ```
 
 - Add color codes to text (uses color names)
 ``` python
 >>> from incolor import incolor, Color
->>> incolor(Color.blue, 'I\'m blue da ba dee')
+>>> incolor('I\'m blue da ba dee', fg=Color.blue)
 "\x1b[34mI'm blue da ba dee\x1b[0m"
 ```
 
 - Join multiple arguments in colorful manner
 ``` python
->>> incolor(Color.brwhite, 'First', 2, 'third', list())
+>>> incolor('First', 2, 'third', list(), fg=Color.brightwhite)
 '\x1b[315mFirst 2 third []\x1b[0m'
 ```
 
-- Color print function (behaves exactly like `print(incolor(...))` ) (__the is yellow here__)
+- Color print function (behaves exactly like `print(incolor(...))`)
 ``` python
->>> cprint(3, 'Yellow: 🚢', 123, 'yay')
+>>> cprint('Yellow: 🚢', 123, 'yay', fg=3)
 Yellow: 🚢 123 yay
 ```
 
```

Comment:
Gotta add some more examples:
* `fg` parameter
* calling the module from command line.